src/hipo2root.c

```diff
@@ -12,16 +17,22 @@
 #include "../lib/bank_containers.h"
 
 int main(int argc, char **argv) {
-    char *in_filename = NULL;
-    int  run_no = -1;
-
-    if (hipo2root_handle_args_err(hipo2root_handle_args(argc, argv, &in_filename, &run_no),
+    char *in_filename  = NULL;
+    char *out_filename = NULL;
+    int  run_no        = -1;
+    bool use_simul     = false;
+    
+    if (hipo2root_handle_args_err(hipo2root_handle_args(argc, argv, &in_filename, &run_no, &use_simul),
                                   &in_filename))
         return 1;
-
-    char *out_filename = (char *) malloc(128 * sizeof(char));
-    sprintf(out_filename, "../root_io/banks_%06d.root", run_no);
-
+    
+    out_filename = (char *) malloc(128 * sizeof(char));
+    if(!use_simul){
+        sprintf(out_filename, "../root_io/banks_%06d.root", run_no);
+    } else{
+        sprintf(out_filename, "%s.root", in_filename);
```

Comment:
While I agree that my directory convention is awful, its use should be consistent. Please update this to use the `root_io` folder, right now its writing everything to the execution folder.

Reply:
I do agree on keeping a uniform directory convention for everything; but, at this moment, I'm constantly working with different files of simuls so it is unconvenient to use your directory convention for this purpose :/

Reply:
How so? In any case, if it's a problem to keep the convention for your personal use then maybe you can keep the repo with the convention while your local version has the change. I can merge the pull request with this change if you really need it but I'll probably end up changing it in a commit anyway out of habit.

Reply:
To keep the format of the main branch I'll use your format. I'll keep another format for myself in the meantime :)

Reply:
Substited this system for the simul run system.